metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
```

Comment:
Before we parse html and match this only on meta content, we can't just use `[^/]`. Too much corner cases for regex match on plain html :(